src/Sql/Sql.Test/ScenarioTests/SqlTestsBase.cs

```diff
@@ -140,6 +142,34 @@ protected GraphRbacManagementClient GetGraphClient(MockContext context)
             return graphClient;
         }
 
+        protected Microsoft.Azure.Graph.RBAC.Version1_6.GraphRbacManagementClient GetGraphClientVersion1_6(MockContext context)
+        {
+            Microsoft.Azure.Graph.RBAC.Version1_6.GraphRbacManagementClient graphClient = context.GetServiceClient<Microsoft.Azure.Graph.RBAC.Version1_6.GraphRbacManagementClient>(TestEnvironmentFactory.GetTestEnvironment());
+            graphClient.BaseUri = TestEnvironmentFactory.GetTestEnvironment().Endpoints.GraphUri;
+            string tenantId = null;
+
+            if (HttpMockServer.Mode == HttpRecorderMode.Record)
+            {
+                tenantId = TestEnvironmentFactory.GetTestEnvironment().Tenant;
+            }
```

Comment:
You want to store the tenant ID in HttpMockServer.Variables in this case, I think

Reply:
Done